core/procedure.cc

```diff
@@ -38,6 +38,7 @@ ProcedureRunConfigDescription()
     nullAccepted = true;
 
     addField("id", &ProcedureRunConfig::id, "ID of run");
+    addField("state", &ProcedureRunConfig::state, "State of the run");
```

Comment:
It is actually a PUT and why would the state not be part of the _run_ configuration?  I can see a run be running, cancelled, paused and so on.